packages/editor-ui/src/modules/settings.ts

```diff
@@ -131,6 +131,15 @@ const module: Module<ISettingsState, IRootState> = {
 				return e;
 			}
 		},
+		async skipOwnerSetup(context: ActionContext<ISettingsState, IRootState>) {
+			try {
+				context.commit('stopShowingSetupPage');
+				await submitSkipOwnerSetup(context.rootGetters.getRestApiContext);
+				return true;
+			} catch (e) {
+				return e;
```

Comment:
nitpick - no point in these returns. I would remove return values

Reply:
open for discussion regarding the FE, but for the BE we decided to go more functional and reduce side-effects i.e. every function should return a value and aim to return what has been changed. Happy to adjust to what you think is best for the FE 👍🏼